nationality-park/style.scss

```diff
@@ -0,0 +1,160 @@
+/*MAIN SETTINGS*/
+$backgroundColor: rgb(30, 30, 30);
+$backgroundColor50: rgba(30, 30, 30, 0.5);
+$accentColor: green;
+$textColor: whitesmoke;
+*{
+    margin: 0;
+    padding: 0;
+    color: $textColor;
+    font-family: Tahoma, sans-serif;
+}
+body{
+    background-color: $backgroundColor;
+}
+.container{
+    width: 100%;
+    height: auto;
+    position: relative;
+}
+.contentWrapper{
+    width: 80%;
+    height: 100%;
+    margin: auto;
+    z-index: 10;
+    position: relative;
+}
+.accentText{
+    background-color: $accentColor;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    display: inline-block;
+    padding: 10px 15px 10px 15px;
+    margin-bottom: 20px;
+}
+h2{
+    border-left: 2px solid $accentColor;
+    padding-left: 20px;
+    margin-top: 50px;
+    margin-bottom: 20px;
+    font-weight: lighter;
+}
+table{
+    background-color: $backgroundColor50;
+    width: 100%;
+    border-collapse: collapse;
+  }
+td{
+    width: 25%;
+    border: 1px solid $backgroundColor;
+    text-align: center;
+  }
+table img{
+    width: 100px;
+  }
+#homeBox{
+    background-size: cover !important;
+    width: 100%;
+    height: 100vh;
+}
+.homeBoxText{
+    padding-top: 20vh;
+    z-index: 200;
+}
+.homeBoxText h1{
+    font-size: 4rem;z-index: 200;
+}
+.btn{
+    background-color: $accentColor;
+    border: none;
+    padding: 10px 15px 10px 15px;
+    transition: 0.25s;
+    display: inline-block;
+    text-decoration: none;
+}
+.btn:hover{
+    filter: brightness(0.8);
+    transition: 0.25s;
+}
+#description{
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    text-align: justify;
+    -webkit-box-orient: vertical;
+    white-space: pre-line;
+    margin-top: 25px;
+    margin-bottom: 25px;
+    max-width: 60%;
+}
+#description-long{
+    display: -webkit-box;
+    white-space: pre-line;
+}
+#gradient{
+    background: linear-gradient(180deg, $backgroundColor50 0%, $backgroundColor 100%);
+    width: 100%;
+    height: 100vh;
+    position: absolute;
+    z-index: 0;
+    top: 0;
+}
+/*PARKS CARD*/
+#parksContainer{
+    margin-top: -250px;
+    z-index: 200;
+    position: relative;
+}
+
+#parksGrid{
+    display: grid; 
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 20px;
+}
+.parkCard{
+    display: grid;
+    height: auto;
+    padding: 20px;
+    transition: 0.25s;
+    background-size: cover;
+    aspect-ratio: 16/9;
+}
+.parkCard:hover{
+    transition: 0.25s;
+    transform: scale(1.05);
+}
+.parkCard > img{
+    width: 50%;
+    margin: auto;
+}
+#nationalParksChart{
+    max-width: 50%;
+}
+
+/*ON PHONE*/
+@media only screen and (max-width: 800px) {
+    #parksGrid{
+        display: grid; 
+        grid-template-columns: 1fr 1fr;
+        gap: 20px;
+    }
+    .homeBoxText h1{
+        font-size: 2rem;z-index: 200;
+    }
+    #nationalParksChart{
+        max-width: 100%;
+    }
+    table{
+        font-size: 0.5rem;
+    }
+    #description{
+        overflow: hidden;
```

Comment:
do not duplicate attributes from the previous lines of styling, once changed don't need to be set again